backend/src/main/java/edonymyeon/backend/image/postimage/PostImageInfoRepository.java

```diff
@@ -6,5 +6,7 @@
 
 public interface PostImageInfoRepository extends JpaRepository<PostImageInfo, Long> {
 
-    List<PostImageInfo> findAllByPostId(Long postId);
+    List<PostImageInfo> findAllByPostId(final Long postId);
+
+    void deleteAllByPostId(final Long postId);
```

Comment:
이 부분 확인해보니까 쿼리가 각각 날라가여 이리네
찾아보니까 직접 쿼리 in절로 하거나 `deleteAllByIdInBatch`으로 삭제하는 방법이 있네요!
확인해보시면 좋을듯여
```suggestion
    @Modifying
    @Query("delete from PostImageInfo pi where pi.post.id=:postId")
    void deleteAllByPostId(@Param("postId") final Long postId);
```

Reply:
아주 좋은데요 🤪